src/WebARKitController.js

```diff
@@ -141,7 +141,8 @@ export default class WebARKitController {
       console.log('Hey, i am here!');
       console.log(this.width);
       //console.log(this.dataHeap);
-      this.webarkit.initTracking(this.id, this.width, this.height)
+      console.log("id is: ", this.id);
+      this.webarkit.initTracking(1, this.width, this.height)
```

Comment:
initTracking need a id greater than zero ( 1 ) otherwise we get the null-pointer-dereference!!